src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -431,7 +432,11 @@ export class SavedObjectsService
       ).toPromise();
 
       this.logger.info('Starting saved objects migrations');
+      const migrationSpan = apm.startSpan('saved_objects', 'migration');
       await migrator.runMigrations();
+      if (migrationSpan) {
+        migrationSpan.end();
+      }
```

Comment:
NIT: `migrationSpan?.end();`  (multiple occurences)

Reply:
for this, we need to turn off `eslint/no-unused-expressions` and use `@typescript-eslint/no-unused-expressions` instead. 
 https://github.com/typescript-eslint/typescript-eslint/issues/1220